src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -21,6 +21,8 @@ internal class Program
 
         private static void Main(string[] args)
         {
+            AppDomain.CurrentDomain.UnhandledException +=
+                new UnhandledExceptionEventHandler(BridgeUnhandledExceptionHandler);
```

Comment:
According to msdn a unhandled exception handler in the default domain gets called for all other domains so we only need to install it once.